src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/StaticWebAssetsIntegrationTest.cs

```diff
@@ -104,6 +104,26 @@ public async Task Publish_WithBuildReferencesDisabled_CopiesStaticWebAssetsToDes
             Assert.FileExists(publish, PublishOutputPath, Path.Combine("wwwroot", "_content", "PackageLibraryTransitiveDependency", "js", "pkg-transitive-dep.js"));
         }
 
+        [Fact]
+        [InitializeTestProject("AppWithPackageAndP2PReference", additionalProjects: new[] { "ClassLibrary", "ClassLibrary2" })]
+        public async Task Publish_NoBuild_CopiesStaticWebAssetsToDestinationFolder()
```

Comment:
Is it useful to add a test that does a `pack --no-build`?

Reply:
There’s already a test for it.